core/frontend/src/store/autopilot.ts

```diff
@@ -53,6 +55,11 @@ class AutopilotStore extends VuexModule {
     )
   }
 
+  get is_safe() {
+    // We can potentially check for external things here
+    return !this.verhicle_armed
+  }
+
```

Comment:
why is this not calling the backend instead?

Reply:
This will give us feedback at 1hz, I don't want yet another thing polling the backends at 1hz for no reason...
I can add to the comment "such as ardupilotManager's /safe endpoint"

Reply:
So why we have the safe endpoint in the backend ?

Reply:
hmm, so why was the backend endpoint added?

Reply:
for subsequent backend-based validations, such in ardupilot_manager.py and whatever else